site/en/docs/extensions/mv3/override/index.md

```diff
@@ -73,26 +69,24 @@ For `PAGE_TO_OVERRIDE`, substitute one of the following:
 - `history`
 - `newtab`
 
-## Tips {: #tips }
+## Best practices {: #tips }
 
-For an effective override page, follow these guidelines:
+* **Make your page quick and small.** <br/>Users expect built-in browser pages to open instantly. Avoid doing things that might take a long
+time, like synchronous fetches of network or database resources.
 
-**Make your page quick and small.**<br/>Users expect built-in browser pages to open instantly. Avoid doing things that might take a long
-time. For example, avoid synchronous fetches of network or database resources.
+* **To avoid user confusion, give your page a title.** <br/> Without a title, the page title defaults to the URL. Specify the title using the `title` tag in your HTML file.
 
-**Include a title in your page.**<br/>Otherwise people might see the URL of the page, which could be confusing. Here's an example of
-specifying the title: `<title>New Tab</title>`
+* **Remember that new tabs give keyboard focus to the address bar first.** Don't rely on keyboard focus defaulting to other parts of the page.
 
-**Don't rely on the page having the keyboard focus.**<br/>The address bar always gets the focus first when the user creates a new tab.
-
-**Don't try to emulate the default New Tab page.**<br/>The APIs necessary to create a slightly modified version of the default New Tab page—with top
+* **Don't copy the default New Tab page.** <br/>The APIs necessary to create a slightly modified version of the default New Tab page—with top
 pages, recently closed pages, tips, a theme background image, and so on—don't exist.
```

Comment:
@AmySteam @oliverdunk Can either of you work out what this is trying to say. The inherited wording makes it sound like the new tab page can't be replaced, but as stated above, it can.

Reply:
This is very odd, because in theory, there's an API for [topsites](https://developer.chrome.com/docs/extensions/reference/topSites/).
I honestly don't know why this would need to be explicitly called out... 🤷🏻‍♀️
@oliverdunk over to you!

Reply:
Suggesting some wording. Devlin said pretty much what I expected - this doesn't feel like the place to call out API limitations, but we should at least preserve the idea that we don't want new tab pages which users may be confused about.

```suggestion
* **Make the new tab page your own.** <br/>Avoid creating a new tab page which users may confuse with Chrome's built in new tab page.
```